assets/javascript/app.js

```diff
@@ -0,0 +1,96 @@
+$(document).ready(function(){
+  $("startButton").click(function(){
+  console.log("startbutton clicked.");
```

Comment:
The timer starts when the page is loaded so I would add the start timer function to the click button.